cli/build.go

```diff
@@ -19,14 +19,14 @@ import (
 	"time"
 
 	"github.com/johnewart/archiver"
+	"github.com/johnewart/narwhal"
```

Comment:
ideally we should add libraries to our own org so folks on the team can help maintain it.

Reply:
Sure, I'll move it in a separate PR :+1: